addon/authenticators/oidc.js

```diff
@@ -13,12 +15,47 @@ import { TrackedObject } from "tracked-built-ins";
 import config from "ember-simple-auth-oidc/config";
 import getAbsoluteUrl from "ember-simple-auth-oidc/utils/absolute-url";
 
+const camelizeObjectKeys = (obj) => {
+  Object.keys(obj).forEach((key) => {
+    obj[camelize(key)] = obj[key];
+    delete obj[key];
+  });
+  return obj;
+};
```

Comment:
```suggestion
const camelizeObjectKeys = (obj) =>
  Object.entries(obj).reduce((newObj, [key, value]) => {
    newObj[camelize(key)] = value;
  }, {});
```

Reply:
And to prevent another dependency and assuming the response is kebab case:
```js
const camelize = s => s.replace(/_./g, x=>x[1].toUpperCase())
```

Reply:
:+1: I think it's safe to assume that the endpoint delivers snake_cased keys. [oidc spec](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata)